src/oidcop/oauth2/add_on/dpop.py

```diff
@@ -112,36 +114,35 @@ def post_parse_request(request, client_id, endpoint_context, **kwargs):
     if not _dpop.key:
         _dpop.key = key_from_jwk_dict(_dpop["jwk"])
 
-    _jkt = str(_dpop.key.thumbprint("SHA-256"))
-    try:
-        endpoint_context.cdb[client_id]["dpop_jkt"][_jkt] = _dpop.key
-    except KeyError:
-        endpoint_context.cdb[client_id]["dpop_jkt"] = {_jkt: _dpop.key}
-
     # Need something I can add as a reference when minting tokens
-    request["dpop_jkt"] = _jkt
+    request["dpop_jkt"] = as_unicode(_dpop.key.thumbprint("SHA-256"))
     return request
 
 
 def token_args(endpoint_context, client_id, token_args: Optional[dict] = None):
```

Comment:
token_args is at the same time the name of the function and the name of an argument.
We should avoid naming collision, do you agree?

Reply:
Yep!